nucliadb_utils/src/nucliadb_utils/storages/s3.py

```diff
@@ -356,6 +356,7 @@ def __init__(
         verify_ssl: bool = True,
         use_ssl: bool = True,
         region_name: Optional[str] = None,
+        kms_key_id: Optional[str] = None,
```

Comment:
Can you move this to be the last argument? This is in nucliadb_utils which is shared with other repos, and I don't know if someone might be calling this function relying on argument order. I mean, it would be a terrrible idea to do so, but someone might be doing it, so just in case.

Reply:
Sure, done!